src/strategies/mantra-dao/index.ts

```diff
@@ -0,0 +1,76 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'Spaceforce';
```

Comment:
This should be your github username
```suggestion
export const author = 'spaceforce-dev';
```

Reply:
Ok will do